CMakeLists.txt

```diff
@@ -98,13 +98,13 @@ endif()
 # Subdirectories #
 ##################
 
+if(DPDK_TARGET)
```

Comment:
Instead of basing the dependency on `DPDK_TARGET`, I would recommend implementing a `WITH_KRNLMON` conditional. krnlmon support is not inherently a DPDK property.

Just add the following earlier in the file:
```
if(DPDK_TARGET)
    set(WITH_KRNLMON ON)
endif()
```

Reply:
Good idea, done